formatTest/typeCheckedTests/expected_output/oo.re

```diff
@@ -30,28 +29,27 @@ let tmp = {
 /**
  * Comment on stackWithAttributes.
  */
```

Comment:
I agree it doesn't look as good, but if we don't like the look, then we should have written the single line comment inside the braces. (I understand that's not an option for converting from ML).

Did you have to include special logic to introduce the new behavior? I think the old behavior was better because you could choose which of the two styles you wanted, and then the formatter would preserve it.